catalystwan/models/configuration/feature_profile/sdwan/application_priority/traffic_policy.py

```diff
@@ -538,6 +538,36 @@ class SlaClassAction(BaseModel):
         default=None, validation_alias="slaClass", serialization_alias="slaClass", description="slaClass"
     )
 
+    @staticmethod
+    def from_params(
+        sla_name: Optional[UUID] = None,
+        preferred_color: Optional[List[TLOCColor]] = None,
+        preferred_color_group: Optional[UUID] = None,
+        preferred_remote_color: Optional[List[TLOCColor]] = None,
+        remote_color_restrict: Optional[bool] = None,
+        strict: Optional[bool] = None,
+        fallback_to_best_path: Optional[bool] = None,
+    ) -> "SlaClassAction":
+        action = SlaClassAction()
+        action.sla_class = []
+        if sla_name is not None:
+            action.sla_class.append(SlaClass(sla_name=RefIdItem.from_uuid(sla_name)))
+        if preferred_color:
+            action.sla_class.append(SlaClass(preferred_color=Global[List[TLOCColor]](value=preferred_color)))
+        if preferred_color_group is not None:
+            action.sla_class.append(SlaClass(preferred_color_group=RefIdItem.from_uuid(preferred_color_group)))
+        if preferred_remote_color is not None:
+            action.sla_class.append(
+                SlaClass(preferred_remote_color=Global[List[TLOCColor]](value=preferred_remote_color))
+            )
+        if remote_color_restrict is not None:
+            action.sla_class.append(SlaClass(remote_color_restrict=Global[bool](value=remote_color_restrict)))
+        if strict is not None:
+            action.sla_class.append(SlaClass(strict=Global[bool](value=strict)))
+        if fallback_to_best_path is not None:
+            action.sla_class.append(SlaClass(fallback_to_best_path=Global[bool](value=fallback_to_best_path)))
```

Comment:
```suggestion
        if sla_name:
            action.sla_class.append(SlaClass(sla_name=RefIdItem.from_uuid(sla_name)))
        if preferred_color:
            action.sla_class.append(SlaClass(preferred_color=Global[List[TLOCColor]](value=preferred_color)))
        if preferred_color_group:
            action.sla_class.append(SlaClass(preferred_color_group=RefIdItem.from_uuid(preferred_color_group)))
        if preferred_remote_color:
            action.sla_class.append(
                SlaClass(preferred_remote_color=Global[List[TLOCColor]](value=preferred_remote_color))
            )
        if remote_color_restrict:
            action.sla_class.append(SlaClass(remote_color_restrict=Global[bool](value=remote_color_restrict)))
        if strict:
            action.sla_class.append(SlaClass(strict=Global[bool](value=strict)))
        if fallback_to_best_path:
            action.sla_class.append(SlaClass(fallback_to_best_path=Global[bool](value=fallback_to_best_path)))
```

Reply:
NIT: Can just check for truthiness

Reply:
We can make this builder simpler, however I opted for more control to user eg. : `strict` can be not inserted at all (when None) it can be `False` or `True`.